src/libraries/System.Net.Sockets/src/System/Net/Sockets/UnixDomainSocketEndPoint.cs

```diff
@@ -124,12 +125,23 @@ public override string ToString()
             }
         }
 
+        public override bool Equals([NotNullWhen(true)] object? obj)
+            => obj is UnixDomainSocketEndPoint ep && MemoryExtensions.SequenceEqual<byte>(_encodedPath, ep._encodedPath);
+
+        public override int GetHashCode()
+        {
+            HashCode hash = default;
+            hash.AddBytes(_encodedPath);
+            return hash.ToHashCode();
+        }
```

Comment:
Why use _encodedPath rather than _path, e.g.
```suggestion
        public override bool Equals([NotNullWhen(true)] object? obj) =>
            obj is UnixDomainSocketEndPoint ep && _path == obj._path;

        public override int GetHashCode() => _path.GetHashCode();
```
?